### PR TITLE
Added loading animation.

### PR DESCRIPTION
**Pull request by stepek**

### CODE REVIEW

**Comment by thayashi** [comment]
Code itself is no problem, could you please change the animation style to "three-bounce"?

File: app/js/views/sampleView.js [hunk 1]
```diff
@@ -25,6 +27,9 @@ export default class SampleView extends View {
     super(options);
     this.app = options.app;
 
+    this.$el.html(loaderTemplate());
+    setTimeout(() => this.render(), 2000); // this was added for showing loading.
```
Comment (thayashi) [inline app/js/views/sampleView.js]: Also, It might be good to have config for this timeout value to be configurable

Comment (stepek) [inline app/js/views/sampleView.js]: This timeout is only for showing how it works.

Comment (thayashi) [inline app/js/views/sampleView.js]: Oh, it's my misunderstanding, please ignore  my comment.

**Comment by stepek** [comment]
Changed animation style.

**Pull request merged**